app/src/Homepage/Homepage-components.test.tsx

```diff
@@ -1,14 +1,17 @@
 import { render, screen } from '@testing-library/react';
 import { LevelTab } from './Homepage-components';
+import { BrowserRouter as Router } from 'react-router-dom';
```

Comment:
Can you remove space between these? Ex. {render, screen}. The TS style guide removes those spaces